cnvlib/cnary.py

```diff
@@ -83,7 +83,6 @@ def by_gene(self, ignore=params.IGNORE_GENE_NAMES):
             Pairs of: (gene name, CNA of rows with same name)
         """
         ignore += params.ANTITARGET_ALIASES
-        start_idx = end_idx = None
```

Comment:
Removed this line because it's never used—both `start_idx` and `end_idx` are initialised again below